cetloxblock/cetloxblock.pyproj

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
When you make a `.gitignore` file, this should also be ignored if it isn't already, i.e. `*.pyproj` files.